api/src/main/java/jakarta/faces/context/PartialResponseWriter.java

```diff
@@ -130,6 +130,8 @@ public void startDocument() throws IOException
         // use writePreamble(...)
         //_wrapped.write("<?xml version=\"1.0\" encoding=\"utf-8\"?>");
```

Comment:
remove this commented out line?

Reply:
Maybe I should also use writePreamble()? 

I'll update the PR.

Reply:
Need to make one more change